CRM/Admin/Form/SettingTrait.php

```diff
@@ -176,14 +176,15 @@ protected function getSettingsOrderedByWeight() {
    */
   protected function addFieldsDefinedInSettingsMetadata() {
     $this->addSettingsToFormFromMetadata();
-    $settingMetaData = $this->getSettingsMetaData();
+    $settingsMetaData = $this->getSettingsMetaData();
```

Comment:
This is just renamed as it confused me that it is many settings, not just the one being iterated